Update 2022-01-25-Related-Projects.md (#1086)
jamulussoftware/jamulussoftware.github.io: _posts/2022-01-25-Related-Projects.md

The [Melomax website](https://melomax.live) offers private servers groups can use with Jamulus client software.

## Running a Jamulus server on a Raspberry Pi 

> ## Measuring transmission quality on Jamulus

Reviewer: I think this should be on one line but shortened.

Author: I have shortened the headline

> ## Running a Jamulus server on a Raspberry Pi and how to measure the transmission quality on Jamulus

Reviewer: ```suggestion
## Measuring transmission quality of Jamulus
```

> This project describes how to use a Raspberry Pi as a Jamulus client.

Reviewer: ```suggestion
The "MusizierenUeberDasInternet" project describes how to use a Raspberry Pi as a Jamulus client.
```

Reviewer: Maybe it's worth making clear that it's scientific work.

Author: Good idea. I will add the following sentence:
The project was developed by students of electrical engineering and computer science at the FH Aachen University of Applied Sciences.

It also examines the influence of the Jamulus settings on the transmission quality and how this can be measured. This was first tested with a function generator and an oscilloscope. A device was then developed with which the transmission errors can be measured easily and automatically. Circuit diagrams, layout and software for this device are described and made available.
The documentation is partly in German and partly in English.

> I hope that some Jamulus users will find our project helpful and interesting. Therefore I would like to make it generally available.

Reviewer: ```suggestion
```

I'd rather not add an "I" here since the author is not visible by default on this page.

> More information about our project can be found here:

Reviewer: ```suggestion
More information about the project can be found here:
```
